src/constants.js

```diff
@@ -71,4 +71,10 @@ export default class Constants {
    */
   static WEBHOOK_DOES_NOT_MATCH =
     'Webhook received did not originate from EasyPost or had a webhook secret mismatch.';
+
+  /**
+   * The template for the error message when a pagination list has reached its end
+   * @type {string}
+   */
```

Comment:
thought: I don't love that we decided to document every constant string. They should be self explanatory from the name and string. Lots of extra unnecessary comments imo